digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -418,7 +442,43 @@ static boolean isDeterministicClientException(Exception ex)
                     return statusCode >= 400 && statusCode < 500;
             }
         }
-        else if (ex instanceof TDClientException) {
+        return isFailedBeforeSendClientException(ex);
+    }
+
+    static boolean isNeedNotRetryException(Exception ex)
```

Comment:
Not serious, but method name may be difficult to understand. If possible rename is better.

Reply:
I fixed to `isDeterministicException` opposite to `isDeterministicClientException` because it includes non-client(authentication) but deterministic. Please tell me if you have better idea.
https://github.com/treasure-data/digdag/pull/1362/commits/972d0b788f7c0ac101c9f9453472fc4ffbbb6db6